### ISSUE TITLE
Automatically fetch ISO/IEC references when used

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/asciidoctor-iso#63]
Just like how we do it in asciidoctor-rfc, we want to automatically fetch ISO and IEC references (title, URI, edition) when it is cited.

Thoughts?

**Comment by opoudjis** [comment]
Is there an online cache of them like for RFC?

**Comment by ronaldtse** [comment]
It doesn't seem like it -- maybe someone should make a gem to search and fetch a certain standard. For example, ISO 9001 is https://www.iso.org/standard/62085.html .

**Comment by opoudjis** [comment]
Screenscrape the ISO shop. Ugh.

This can be done, but please get someone else to do it. (I'll keep the job assigned to me, but it'll be very low priority for me.) Once the data is fetched, yes, asciidoctor-iso can fish in it.

**Comment by ronaldtse** [comment]
Agree with you :wink:

**Comment by ronaldtse** [comment]
This issue now depends on the implementation of isobib https://github.com/riboseinc/isobib

**Comment by ronaldtse** [comment]
@andrew2net this will be for you 👍

**Comment by opoudjis** [comment]
It depends on https://github.com/riboseinc/isodoc/issues/22: the gem currently assumes references are preformatted, and once isobib references land in asciidoctor-iso, that gem can generate the appropriate XML — but isodoc will need to be updated to render them.

**Comment by opoudjis** [comment]
Has been done in isobib gem.
